M2/Macaulay2/tests/normal/doc-links.m2

```diff
@@ -6,6 +6,7 @@ assert ( match (
 	  "<span><a title=\"an example Macaulay2 package\" href=\".*/FirstPackage/html/index.html\">FirstPackage</a> -- an example Macaulay2 package</span>",
 	  html TOH "FirstPackage::FirstPackage"))
 needsPackage "FirstPackage"
+if not FirstPackage#?"raw documentation database" then end
```

Comment:
I'm still not super satisfied with this solution.  We don't want the test to fail if we're not building the documentation, but we would want it to fail if we did build the documentation!

Reply:
Possible a better check would be to check if the database file exists, and then exit if it doesn't.  If it exists, then we'll check to see if it actually works or not.

Reply:
You can also install FirstPackage under /tmp and test it through that database.

Reply:
Ooh, that's a good idea!  I'll do that instead.